### PR TITLE
[Config] Use RectorConfig::configure() on tests

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

File: tests/Skipper/Skipper/config/single_skipped_rule_config.php [hunk 1]
```diff
@@ -6,11 +6,6 @@
...
-    $rectorConfig->rules([
-        InlineConstructorDefaultToPropertyRector::class,
-        RemoveUnusedPromotedPropertyRector::class,
-    ]);
-
-    $rectorConfig->skip([InlineConstructorDefaultToPropertyRector::class]);
-};
+return RectorConfig::configure()->withSkip([InlineConstructorDefaultToPropertyRector::class])->withRules(
+    [InlineConstructorDefaultToPropertyRector::class, RemoveUnusedPromotedPropertyRector::class]
+);
```
Comment (TomasVotruba) [inline tests/Skipper/Skipper/config/single_skipped_rule_config.php]: These should be indented with single call per line.

Comment (samsonasik) [reply tests/Skipper/Skipper/config/single_skipped_rule_config.php]: any ecs config can be applied?

Comment (samsonasik) [reply tests/Skipper/Skipper/config/single_skipped_rule_config.php]: I updated to use new line per-call

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I am merging it ;)

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**